components/classes/experience.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/experience",
```

Comment:
Recommend changing the ID. `https://ns.adobe.com/xdm/classes/experience` is very generic. Could you come up with something that is unique to your use case?